lib/model/CModelDetailsView.cc

```diff
@@ -159,7 +159,7 @@ bool CModelDetailsView::contains(const TStrSet& terms, const std::string& key) {
 }
 
 bool CModelDetailsView::hasByField() const {
-    return (this->base().isPopulation()
+    return !(this->base().isPopulation()
```

Comment:
wouldn't it be easier to switch the conditional expressions?

`!` is always hard to parse and probably the root of this regression?

Reply:
obviously hard, I overlooked the brackets. Maybe ` == false` - I think we recently discussed to use this in favor of `!`

Reply:
Do you mean changing this method into `hasNoByField` and then inverting the conditionals where it's called from? I'd rather not do that. But we could certainly change the function to the format `(...).empty() == false`. Do you prefer this?